src/main/java/net/fabricmc/loader/impl/util/ManifestUtil.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.loader.impl.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.jar.Attributes.Name;
+import java.util.jar.Manifest;
+
+public final class ManifestUtil {
+	public static final Name NAME_MAPPING_NAMESPACE = new Name("Fabric-Mapping-Namespace");
```

Comment:
Are these constants used now?

Reply:
hmm the whole class is unused for now..

Reply:
The file is actually still used through a refactor (moved Knot's manifest reading to it too), but now without the constants.